packages/web/src/questionnaire/trackSiret.ts

```diff
@@ -37,8 +37,9 @@ export const siret = {
       id: 'search-siret',
       value: { ...dataTarget },
       title: { fr: 'SIRET' },
-      label: { fr: 'Renseignez le SIRET de votre entreprise (14 chiffres)' },
-      placeholder: { fr: 'ex : 830 141 321 00034' },
+      // label: { fr: 'Renseignez le SIRET de votre entreprise (14 chiffres)' },
+      placeholder: { fr: 'Votre numéro SIRET (14 chiffres)' },
+      hint: { fr: `Besoin d'aide pour retrouver votre SIRET ?&nbsp;<a href="https://annuaire-entreprises.data.gouv.fr/" target="_blank">Cliquez ici</a>` },
```

Comment:
suggestion:`&nbsp;` is a non-breakable space, maybe it can be better to have a breakable space   (?)

![image](https://github.com/betagouv/transition-ecologique-entreprises-widget/assets/1521862/7400453a-a20b-4932-8e73-b3e12e7d9da2)